src/index.js

```diff
@@ -26,6 +30,9 @@ const defaultConfig = {
 };
 
 module.exports = function workerLoaderPlugin(userConfig = null) {
+    if (userConfig) {
+        userConfig.webWorkerPattern = userConfig?.webWorkerPattern || userConfig?.pattern || null;
+    }
```

Comment:
I think we could release this as a major version so breaking backwards compatibility would be ok. I am thinking that having `pattern` and `*WorkerPattern` could get confusing for new users.

Reply:
Ok, I'll remove these few lines then 😄